middlewares/accesslog/json_log_formatter_test.go

```diff
@@ -0,0 +1,135 @@
+package accesslog
+
+import (
+	"bytes"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestJSONLogFormatter(t *testing.T) {
+	jlf, err := newJSONLogFormatter(&types.AccessLog{
+		TimeFormat: commonLogTimeFormat,
+		CoreFields: []string{
+			"StartUTC:StartUTC",
+			"Duration:Duration",
+			"FrontendName:thefrontend",
+			"BackendName:BackendName",
+			"BackendURL:BackendURL",
+			"OriginDuration:OriginDuration",
+			"OriginContentSize:OriginContentSize",
+			"RequestAddr:RequestAddr",
+			"RequestMethod:RequestMethod",
+			"RequestPath:RequestPath",
+			"RequestProtocol:RequestProtocol",
+			"OriginStatus:OriginStatus",
+			"DownstreamContentSize:DownstreamContentSize",
+			"RequestCount:RequestCount",
+			"ClientHost:ClHost",
+			"ClientPort:ClPort",
+			"ClientUsername:ClUsername",
+		},
+		RequestHeaders:            []string{"User-Agent: user_agent", "Referer: referrer"},
+		OriginResponseHeaders:     []string{"Server: upstream_http_server"},
+		DownstreamResponseHeaders: []string{"Location: sent_http_location"},
+	})
+
+	assert.True(t, err == nil)
+
+	buf := &bytes.Buffer{}
+	jlf.Write(buf, fixtureLogDataTable(12345))
+	s := buf.String()
+	assert.Equal(t,
+		`{"StartUTC":"`,
+		s[:13])
+	assert.Equal(t,
```

Comment:
Would it then be possible to have an assertion to verify that whatever is at the position of the timestamp matches the format we're expecting?